test/boulder-config.json

```diff
@@ -335,6 +335,11 @@
     "dbConnectFile": "test/secrets/cert_checker_dburl"
   },
 
+  "expiredAuthzPurger": {
+    "dbConnectFile": "test/secrets/purger_dburl",
+    "gracePeriod": "168h"
+  },
+
```

Comment:
@rolandshoemaker - should this be in both `boulder-config-next` and `boulder-config`? (Reminds me I need to PR a clarifying line for this in CONTRIBUTING.md)